Library/Format.swift

```diff
@@ -105,6 +105,30 @@ public enum Format {
       ?? (symbol + "\(amount)")
   }
 
+  public static func attributedCurrency(_ amount: Double,
+                                        country: Project.Country,
+                                        omitCurrencyCode: Bool = false,
+                                        defaultAttributes: String.Attributes = [:],
+                                        superscriptAttributes: String.Attributes = [:],
+                                        env: Environment = AppEnvironment.current) -> NSAttributedString? {
+    let symbol = currencySymbol(forCountry: country, omitCurrencyCode: omitCurrencyCode, env: env)
+    let config = NumberFormatterConfig.defaultCurrencyConfig
+      |> NumberFormatterConfig.lens.locale .~ env.locale
+      |> NumberFormatterConfig.lens.currencySymbol .~ symbol
+      |> NumberFormatterConfig.lens.maximumFractionDigits .~ 2
+
+    guard let formatter = NumberFormatterConfig.cachedFormatter(forConfig: config)
+      as? AttributedNumberFormatter else { return nil }
+
+    _ = formatter
```

Comment:
Pity about having to do this discarded value 🤔  Only real alternative I could come up with was:

```swift
    return (
      formatter
        |> \.defaultAttributes .~ defaultAttributes
        |> \.currencySymbolAttributes .~ superscriptAttributes
        |> \.decimalSeparatorAttributes .~ superscriptAttributes
        |> \.fractionDigitsAttributes .~ superscriptAttributes
      )
      .attributedString(for: amount)
```
😅